Index: synctex/synctex_parser_utils.c
===================================================================
--- synctex.orig/synctex_parser_utils.c	2010-05-23 18:49:35.000000000 +0200
+++ synctex/synctex_parser_utils.c	2010-05-23 19:04:27.000000000 +0200
@@ -142,7 +142,7 @@
 
 /*  Compare two file names, windows is sometimes case insensitive... */
 synctex_bool_t _synctex_is_equivalent_file_name(const char *lhs, const char *rhs) {
-#	if SYNCTEX_WINDOWS
+#	ifdef SYNCTEX_WINDOWS
     /*  On Windows, filename should be compared case insensitive.
 	 *  The characters '/' and '\' are both valid path separators.
 	 *  There will be a very serious problem concerning UTF8 because
@@ -174,7 +174,7 @@
 	if(!strlen(name)) {
 		return synctex_NO;
 	}
-#	if SYNCTEX_WINDOWS
+#	ifdef SYNCTEX_WINDOWS
 	if(strlen(name)>2) {
 		return (name[1]==':' && SYNCTEX_IS_PATH_SEPARATOR(name[2]))?synctex_YES:synctex_NO;
 	}
Index: synctex/synctex_parser_utils.h
===================================================================
--- synctex.orig/synctex_parser_utils.h	2010-05-23 18:49:35.000000000 +0200
+++ synctex/synctex_parser_utils.h	2010-05-23 19:03:30.000000000 +0200
@@ -55,7 +55,7 @@
 extern "C" {
 #endif
 
-#	if _WIN32
+#	ifdef _WIN32
 #		define SYNCTEX_IS_PATH_SEPARATOR(c) ('/' == c || '\\' == c)
 #	else
 #		define SYNCTEX_IS_PATH_SEPARATOR(c) ('/' == c)
